align_system/configs/adm/oracle.yaml

```diff
@@ -1,7 +1,9 @@
 instance:
   _target_: align_system.algorithms.oracle_adm.OracleADM
 
+inference_kwargs:
   probabilistic: false
   upweight_missing_kdmas: false
```

Comment:
`probabilistic` and `upweight_missing_kdmas` are currently set up as instance `kwargs` (arguments of the constructor not `choose_action`). It might make sense to change them to inference kwargs, but I'm not sure this config is quite right as is (though works for this configuration because the config values match the defaults).

Reply:
Thanks, I've fixed it